crates/bindings-csharp/Runtime/Internal/Bounds.cs

```diff
@@ -18,7 +18,6 @@ public interface IBTreeIndexBounds
 }
 
 public readonly struct Bound<T>(T min, T max)
-    where T : IEquatable<T>
```

Comment:
Why were these bounds removed? Does the codegen fail to implement `IEquatable` somewhere, e.g. for plain `enum`s? I haven't looked at C# codegen for plain `enum`s at all.

Reply:
And you can't implement interfaces for `enum`s, right. Ah well.